src/main/java/cms/gov/madie/measure/services/VersionService.java

```diff
@@ -110,5 +120,14 @@ protected void getNextVersion(Measure measure, String versionType) {
     } else if (VERSION_TYPE_PATCH.equalsIgnoreCase(versionType)) {
       measure.getVersion().setRevisionNumber(measure.getVersion().getRevisionNumber() + 1);
```

Comment:
I believe versioning of a measure should happen based on the maxValueof that particular version from the family.

Reply:
@RohitKandimalla I have added searching the max version number from the family. Please let me know if it's good. Thank you Rohit!